samples/transaction.js

```diff
@@ -122,7 +119,7 @@ function readWriteTransaction(instanceId, databaseId, projectId) {
   const transferAmount = 200000;
   const minimumAmountToTransfer = 300000;
 
-  database.runTransaction((err, transaction) => {
+  database.runTransaction(async (err, transaction) => {
```

Comment:
Having an async callback is very weird.  Should we just make the top level method this is under async instead?